syntax = "proto3";

message AppMarketDataOutboxMessage {
  string stock_symbol = 1;
  double latest_price = 2;
  string timestamp = 3;
}

message AppMarketDataOutboxMessageList {
  repeated AppMarketDataOutboxMessage messages = 1;
}